docs/reference/classes.rst

```diff
@@ -76,18 +78,23 @@ Object
 A physical object.
 This class subclasses `OrientedPoint`, adding a variety of properties including:
 
-* :prop:`width` and :prop:`length` to define the bounding box of the object;
+* :prop:`width`, :prop:`length`, and height to define the dimensions of the object;
+* :prop:`shape` to define the shape of the object;
 * :prop:`allowCollisions`, :prop:`requireVisible`, and :prop:`regionContainedIn` to control the built-in requirements that apply to the object;
 * :prop:`behavior`, specifying the object's :term:`dynamic behavior` if any;
 * :prop:`speed`, :prop:`velocity`, and other properties capturing the dynamic state of the object during simulations.
 
 The built-in requirements applying to each object are:
 
 * The object must be completely contained within its :term:`container`, the region specified as its :prop:`regionContainedIn` property (by default the entire :term:`workspace`).
-* The object must be visible from the ego object, unless its :prop:`requireVisible` property is set to `False`.
+* The object must be visible from the ego object if the :prop:`requireVisible` property is set to `True` (default value `False`).
 * The object must not intersect another object (i.e., their bounding boxes must not overlap), unless either of the two objects has their :prop:`allowCollisions` property set to `True`.
 
-.. autoclass:: scenic.core.object_types.Object
+.. versionchanged::3.0
```

Comment:
```suggestion
.. versionchanged:: 3.0
```
(This space matters: the line is currently being parsed as an RST comment, not a directive.)